.github/workflows/coverage.yaml

```diff
@@ -121,7 +121,9 @@ jobs:
         shell: bash
         run: |
           source ./dozer-tests/python_udf/virtualenv.sh
-          cargo test --features snowflake,ethereum,kafka,python,mongodb --no-fail-fast
+          apt-get update && apt-get install -y npm
+          (cd ./dozer-tests/wasm_udf/assemblyscript && npm install && npm run asbuild)
```

Comment:
This is built here, but not used yet. It's here to be used as a resource for tests that want to run wasm_udfs.